Fix Ethereum transfer link with specified address

nichechristie opened the pull request:
Fixes #45

Add Solidity version and import statement to `Transfers.sol`.

* Add Solidity version pragma `^0.8.17`.
* Add import statement for `IERC20` from OpenZeppelin contracts.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/coinbase/commerce-onchain-payment-protocol/pull/48?shareId=83649476-9269-441d-ad38-9bb3a16a1a66).

nichechristie committed:
Fix Ethereum transfer link with specified address

Fixes #45

Add Solidity version and import statement to `Transfers.sol`.

* Add Solidity version pragma `^0.8.17`.
* Add import statement for `IERC20` from OpenZeppelin contracts.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/coinbase/commerce-onchain-payment-protocol/issues/45?shareId=XXXX-XXXX-XXXX-XXXX).

cb-heimdall commented:
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>1</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>2</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>
<!-- CODEOWNERS & Bar Raisers -->
<!-- Power Rangers -->
<!-- Links -->

nichechristie committed:
Improve documentation and add comments

Add detailed usage examples, setup instructions, and test running steps to `README.md`.

Add `CONTRIBUTING.md` with guidelines for contributing, reporting issues, requesting features, and coding standards.

Add `CODE_OF_CONDUCT.md` with community guidelines, reporting unacceptable behavior, and enforcement details.

Add inline comments to `contracts/interfaces/ITransfers.sol` to explain complex logic and improve readability.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/Nichebiche/commerce-onchain-payment-protocol?shareId=XXXX-XXXX-XXXX-XXXX).

nichechristie committed:
Enhance code quality by adding unit tests and implementing a linter

Update `README.md` and `CONTRIBUTING.md` to reflect the addition of unit tests and implementation of a linter.

* **README.md**
  - Add sections for unit tests and linter with instructions on how to run them.

* **CONTRIBUTING.md**
  - Add instructions for running unit tests and using the linter.

* **contracts/interfaces/IERC7597.sol**
  - Remove SPDX-License-Identifier comment.

* **contracts/interfaces/IWrappedNativeCurrency.sol**
  - Remove SPDX-License-Identifier comment.

* **contracts/utils/Sweepable.sol**
  - Remove SPDX-License-Identifier comment.

---

For more details, open the [Copilot Workspace session](https://copilot-workspace.githubnext.com/Nichebiche/commerce-onchain-payment-protocol?shareId=XXXX-XXXX-XXXX-XXXX).

nichechristie commented:
> # 🟡 Heimdall Review Status
> Requirement	Status	More Info
> Reviews	🟡 `0/1`	
> Denominator calculation

Can you review the code

nichechristie committed:
Merge pull request #2 from Nichebiche/enhance-code-quality

nichechristie committed:
Merge pull request #1 from Nichebiche/improve-documentation

cb-heimdall commented:
Review Error for Nplacentino1 @ 2025-02-24 03:17:28 UTC
User must have write permissions to review

cb-heimdall commented:
Review Error for Nickwittastick23 @ 2025-10-11 03:12:35 UTC
User must have write permissions to review

cb-heimdall commented:
Review Error for Nickwittastick23 @ 2025-10-16 13:29:47 UTC
User must have write permissions to review

Irasone commented:
Pul
